ci/run_test_main.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This is the script used for the run_tests.sh stage in CI
+# for doing unit testing.
+set -ex
+
+# JENKINS-52781 tar function is breaking symlinks
+rm -rf test_results
+mkdir test_results
+# shellcheck source ./.build_vars.sh
```

Comment:
(lint) Couldn't parse this shellcheck annotation. [SC1073]
(lint) Expected "disable". Fix any mentioned problems and try again. [SC1072]